app/javascript/stylesheets/layouts/sidebar.scss

```diff
@@ -20,3 +20,13 @@ $sidebar-width: 72px;
   border-bottom: thin solid theme("colors.gray.300");
   padding: 8px 12px;
 }
+
+.neeto-ui-pane__wrapper {
+  .nui-pane__footer--absolute {
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    padding: 32px 40px;
+    border-top: 1px solid theme("colors.gray.300");
+  }
+}
```

Comment:
Why do we need to do this?

Reply:
I will cross verify if this is duplicate and will remove if not needed.